velox/exec/PlanNodeStats.cpp

```diff
@@ -49,8 +49,8 @@ void PlanNodeStats::addTotals(const OperatorStats& stats) {
 
   peakMemoryBytes += stats.memoryStats.peakTotalMemoryReservation;
 
-  for (const auto& entry : stats.runtimeStats) {
-    customStats[entry.first].merge(entry.second);
+  for (const auto& [name, metric] : stats.runtimeStats) {
```

Comment:
These refactors are nice, but they make it difficult to focus on the substantive changes. Any chance these can be moved into a separate PR?